### PR TITLE
task 2: ast rewrite

### PR DESCRIPTION
**Pull request by migeed-z**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #61310
* #61265
* #61239
* #61238
* #61188
* #61150
* #61093
* #61012
* #60805
* **#60622**

Differential Revision: [D29493747](https://our.internmc.facebook.com/intern/diff/D29493747)

### CODE REVIEW

File: torch/fx/experimental/rewriter.py [hunk 1]
```diff
@@ -64,6 +64,18 @@ def visit_Assert(self, node):
         # a replacement for the original _assert node
         return ast.copy_location(expr_wrapper, node)
 
+    def visit_AnnAssign(self, node):
+        """
+        Swap out Python's AnnAssign with an Assign note where the annotation function is called.
```
Comment (jamesr66a) [inline torch/fx/experimental/rewriter.py:69]: nit: `s/note/node`

**Review by jamesr66a — APPROVED** [review]
image

**Comment by migeed-z** [comment]
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29493747).

**Comment by migeed-z** [comment]
@migeed-z has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29493747).

**Pull request closed without merging**